packages/node/src/store.ts

```diff
@@ -59,7 +59,9 @@ export class Store {
    * Returns the StateChannel instance with the specified multisig address.
    * @param multisigAddress
    */
-  async getStateChannel(multisigAddress: Address): Promise<StateChannel> {
+  public async getStateChannel(
```

Comment:
It's `public` by default. No need to specify unless making it `private`.

Reply:
This is just a preference of always biasing to be explicit about things when possible

Reply:
👍